src/main/java/att/grappa/Element.java

```diff
@@ -994,8 +994,6 @@ public static String canonString(String input)
         StringBuilder strbuf = new StringBuilder(len + 8);
         char[] array = input.toCharArray();
         boolean has_special = false;
-        boolean isHTML = false;
```

Comment:
HTML labels are supported, according to [the documentation](http://www.graphviz.org/content/node-shapes#html) and a quick test that I ran. I'd say this commit should be skipped. Is there anything breaking on your end?

Reply:
You are probably right. What I'm facing with is, that in our project we put a HTML-like attribute value into the "graph" section (Graph.setAttribute() ), and dot claims for an illegal character here.
It there is no problem, where the HTML-like value is quoted.
But there are some things not clear for me:
 1. Why is is required in the original code to disable quoting for HTML-like attributes? Does it matters something in the graphviz side?
 2. Why do we put HTML-like attributes in the "graph" section.

Reply:
HTML attributes are surrounded by `<...>`, for example:

    a [label=<<I>hello</I>>]
    b [label=<
      <B>Longer text</B>
      <BR/>
      <I>Spanning multiple lines</I>
    >]

Reply:
So, in this case `<` and `>` act like the quotes. Using regular quotes works too, but in that case it loses it's special "this is an HTML label" meaning, and using nothing to quote is invalid.

Reply:
It is interesting, as we are using a similar form (for an unknown reason), and dot claims error.
Example:
```
digraph "our content" {
  graph [
    layoutAlgorithmConfig = <layout-algorithm-config><edge-mapping /></layout-algorithm-config>,
...
```
dot error:
```
Error: temp/dot.in: syntax error in line 3 near '>'
```
Maybe it accepts forms like this for a label, but for any other attributes?
If you can recommend me a proper solution for that, I will implement it. Or we just can leave this commit out, as we can solve this issue internally.

Reply:
Okay, now I understand. Quoting with < > works as you have described. Thx! I have reverted this commit.